src/Modules/mpolyquo.jl

```diff
@@ -1,209 +1,155 @@
-# We implement the SubModuleOfFreeModule-level of the module functionality 
-# interface for modules over MPolyQuos.
+const AdmissibleSingularQuoRingElem = Union{<:MPolyQuoRingElem{<:MPolyRingElem{<:FieldElem}}}
```

Comment:
```suggestion
const AdmissibleSingularQuoRingElem = Union{MPolyQuoRingElem{<:MPolyRingElem{<:FieldElem}}}
```
you probably want this

Reply:
Yes, maybe. I presume, this will be subject to change, either way, as we will gather and discuss for which quotient rings we will have a singular side available. That will also be, when the `Union` will actually come to bear.